### PR TITLE
fix(travis): Support timestamp in JenkinsBuildInfo (provided by Travis)

### PR DESCRIPTION
**Pull request by jervi**
(Travis is piggybacking on the Jenkins trigger implementation in Orca)

### CODE REVIEW

**Comment by jervi** [comment]
@spinnaker/reviewers PTAL 🙂

File: orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/JenkinsTrigger.kt [hunk 1]
```diff
@@ -57,7 +57,8 @@ constructor(@param:JsonProperty("name") override val name: String?,
             @param:JsonProperty("result") override val result: String?,
             @param:JsonProperty("artifacts") override val artifacts: List<JenkinsArtifact>?,
             @param:JsonProperty("scm") override val scm: List<SourceControl>?,
-            @param:JsonProperty("building") override var building: Boolean = false)
+            @param:JsonProperty("building") override var building: Boolean = false,
+            @param:JsonProperty("timestamp") val timestamp: Long?)
```
Comment (robfletcher) [inline orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/JenkinsTrigger.kt]: Is it really nullable or will Jenkins always supply one?

**Comment by ajordens** [comment]
Slightly off-topic but `(Travis is piggybacking on the Jenkins trigger implementation in Orca)` ... is there a path to making this not the case? 

Strikes me as strange, is there a reason it has to be this way or is it legacy?

**Comment by jervi** [comment]
It is legacy I think. The Travis implementation was in the beginning only in Igor; a Travis master appeared in the rest of Spinnaker as a Jenkins master named “travis-something”. @gardleopard would know more, I think.

File: orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/JenkinsTrigger.kt (hunk 1 above)
Comment (jervi) [reply orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/JenkinsTrigger.kt]: I don’t really know if Jenkins supplies this field at all... Travis always supplies it.

**Pull request merged**